packages/headless/src/components/WPHead.tsx

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
 import Head from 'next/head';
 import { usePost, useGeneralSettings } from '../api';
+import { EnqueuedStylesheet } from '../types';
+import { trimTrailingSlash } from '../utils';
+
+const WP_URL = trimTrailingSlash(
```

Comment:
This is ripe for a helper/util 😁

Reply:
`WP_URL`, that is

Reply:
Yeah. It was used in three other places before I added that.

Reply:
Can we add a dedicated function for this with a separate PR?

Reply:
Yeah, totally! Sorry, didn't mean for it to be a blocker.